### ISSUE TITLE
Consider adding YAFC - Yet Another Factorio Calculator 

### ISSUE THREAD

**Issue by gepgithubber** [deniszholob/factorio-cheat-sheet#392]
Consider adding YAFC - Yet Another Factorio Calculator  ... a desktop application found at URL below.

https://github.com/ShadowTheAge/yafc/tree/master

**Comment by deniszholob** [comment]
i never tried it
looks like its an installation not a web app, will deter folks from trying maybe
but i dont mind adding a link

**Comment by gepgithubber** [comment]
Yes, it is a Microsoft Windows pc installed application.  Thanks
